src/eigenvalue.cpp

```diff
@@ -388,10 +389,15 @@ int openmc_get_keff(double* k_combined)
   k_combined[0] = 0.0;
   k_combined[1] = 0.0;
 
-  // Make sure we have at least four realizations. Notice that at the end,
-  // there is a N-3 term in a denominator.
+  //Special case for n <=3. Notice that at the end,
+  //there is a N-3 term in a denominator.
```

Comment:
Add a space after the `//` for these comments.

Reply:
Done